types/index.d.ts

```diff
@@ -68,6 +68,7 @@ export class AbstractCaver {
     constructor(provider?: RequestProvider, net?: net.Socket)
 
     version: string
+    currentProvider: RequestProvider
```

Comment:
How about setting this type as `provider` which can be imported from `'./packages/caver-core-requestmanager/src'` like below?

```
import { provider, HttpProvider, IpcProvider, WebsocketProvider } from './packages/caver-core-requestmanager/src'
```

Reply:
oh, I missed the fact which currentProvider type is `not string`. Thanks, That's good catch!!

I modify it.